controllers/errors.controllers.js

```diff
@@ -0,0 +1,10 @@
+const express = require("express");
+const app = express();
+
+app.use((err, request, response, next) => {
+    console.log('err in error handling middleware', err)
+    if(err) {
+        response.status(500).send({msg: 'Internal server error'})
+    }
+    next(err)
+})
```

Comment:
What is going on in this file? This is not how to set up an independent error handling file, refer back to the notes on this. No need to make a new express in here.

Reply:
I wanted to move error handling into another file after I learnt it from lecture this morning.  I had corrected it this afternoon. At least I think so.